editor-components/stats/stats4/stats4.tsx

```diff
@@ -1,91 +1,360 @@
+              value: "Valuable happy clients."
+            },
+            {
+              type: "string",
+              key: "content",
+              displayer: "Content",
+              value: "We have crafted beautiful and engaging web solutions."
+            },
+            {
+              type: "number",
+              key: "stat",
+              displayer: "Stat Value",
+              value: 1750
+            }
+          ]
+        },
+        {
+          type: "object",
+          key: "statItem",
+          displayer: "Stat Item",
+          value: [
+            {
+              type: "string",
+              key: "title",
+              displayer: "Title",
+              value: "Presence in countries."
+            },
+            {
+              type: "string",
+              key: "content",
+              displayer: "Content",
+              value: "We have crafted beautiful and engaging web solutions."
+            },
+            {
+              type: "number",
+              key: "stat",
+              displayer: "Stat Value",
+              value: 50
+            }
+          ]
+        },
+        {
+          type: "object",
+          key: "statItem",
+          displayer: "Stat Item",
+          value: [
+            {
+              type: "string",
+              key: "title",
+              displayer: "Title",
+              value: "Worldwide projects."
+            },
+            {
+              type: "string",
+              key: "content",
+              displayer: "Content",
+              value: "We have crafted beautiful and engaging web solutions."
+            },
+            {
+              type: "number",
+              key: "stat",
+              displayer: "Stat Value",
+              value: 856
+            }
+          ]
+        },
+
+      ]
+    })
+    this.state["componentProps"]["selectedFaqIndex"] = null;
+
     this.addProp({
-      type: "string",
-      key: "bottomCardDescription",
-      displayer: "Bottom Card Description",
-      value: "daily total sales",
+      type: "number",
+      key: "animation-duration",
+      displayer: "Number Animation Duration (ms)",
+      value: 500,
     });
+
+    this.castToObject<STAT[]>("statItems").map((statsData, index) =>
+      this.setComponentState(`number-${index}`, 0)
+    );
+
+    let x = setInterval(() => {
+      this.castToObject<STAT[]>("statItems").map(
+        (statsData: STAT, index: number) => {
+          let statNumber = this.getComponentState(`number-${index}`);
+          if (statNumber != statsData.stat) {
+            this.setComponentState(
+              `number-${index}`,
+              Math.min(
+                statsData.stat,
+                statNumber +
+                Math.ceil(
+                  statsData.stat /
+                  Math.round(this.getPropValue("animation-duration") / 30)
+                )
+              ) || 0
+            );
+          }
+        }
+      );
+    }, 30);
```

Comment:
I couldn't see a clear interval function. we need to clear a interval. otherwise, the interval works infinity